site/en/publish/google-play-games.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Cross-publishing Google Play Games for PC
+metadesc: How to ensure the Play Store will surface your Google Play Games titles listed on PC.
```

Comment:
```suggestion
metadesc: How to ensure the Chrome OS Play Store will surface your Google Play Games titles.
```

I think this meta description needs to be updated too? Here's a pass. Otherwise, it reads like it's for PC, not Chromebooks

Reply:
Yah that works. Feel free to push an update with that